src/game-interface/state-actions.py

```diff
@@ -0,0 +1,143 @@
+# Fireplace-related imports do not appear to be necessary since this function
+# takes a player object
+
+def state_actions(player):
+  # In utils.py, player.hero.power appears to return an object representing
+  # the player's hero power. It does not appear to be explicity declared in
+  # player.py, so this check is similar to that used withing the 
+  # actionable_entities() function
+  if player.hero.power:
+    heropower = player.hero.power
+
+    if heropower.is_usable() and heropower.has_target():
+      # Another attribute defined nowhere else in the API
+      # Unsure what this returns as of yet, storing targets for now
+      hpowactions = heropower.targets
+
+  # Getting the hand directly from the player seems better than passing
+  # in the player's hand into our future function
+  for card in player.hand:
+    if card.is_playable():
+      # This next check appears to be for certain Druid class cards
+      if card.must_choose_one:
+        cardactions = card.choose_cards
+      if card.has_target():
+        cardactions = cardactions + card.targets
+      # The player attributes below appear to only be referenced during the
+      # Mulligan phase of a give game. Again, storing for now
+      if player.choice:
+        choiceactions = player.choice.cards
+
+  # Characters are defined are involved in functions and properties in player.py
+  for character in player.characters:
+    if character.can_attack():
+      attackactions = character.targets
```

Comment:
In its current state, this is true. I was focused on looking at one card's possible actions and only seemed to remember that the later checks were inside a loop for each card in the bot's hand for only one check. I will make this change and push it to my branch in order to update this pull request.